### PR TITLE
Update social icons in header and footer

### PR DESCRIPTION
**Pull request by mosabua**
* Adding Linkedin since we use that most
* Removing twitch icon with broadcast link since we have separate page and are mostly on youtube
* Moving Twitter to end of list, could potentially also delete

### CODE REVIEW

**Review by dain — APPROVED** [review]
IMO, we could drop twitter since we don't really use it anymore.  But I'm fine either way

File: _includes/header.html [hunk 1]
```diff
@@ -75,7 +72,10 @@ <h2>Do you ❤️ Trino? Give us a 🌟 on GitHub <i class="fab fa-github"></i><
             <a class="nav-link" href="/slack.html"><i class="fab fa-slack"></i></a>
           </li>
           <li class="nav-item">
-            <a class="nav-link" href="/broadcast/"><i class="fab fa-twitch"></i></a>
```
Comment (bitsondatadev) [inline _includes/header.html:78]: lol I was wondering when this would get the boot! lol

**Comment by bitsondatadev** [comment]
LGTM

**Comment by bitsondatadev** [comment]
> IMO, we could drop twitter since we don't really use it anymore. But I'm fine either way

I am also off Twitter. I did get us a few trino/trinodb spots on a few data centric mastodon accounts.

**Pull request merged**